modules/mod_ginger_collection/templates/collection/list/list-item.tpl

```diff
@@ -1,6 +1,6 @@
 {% with item._source as record %}
     <li class="list__item--beeldenzoeker {{ extraClasses }}">
-        <a href="{% block link %}{% url collection_object database=item._type object_id=record.priref %}?query_id={{ query_id }}&total={{ items.total }}&current={{ current|collection_query_offset:pagelen:page }}{% endblock %}"{% if link_target %} target="{{ link_target }}"{% endif %}>
+        <a href="{% block link %}{% url collection_object database=item._type object_id=record.priref %}{% if query_id %}?query_id={{ query_id }}&total={{ items.total }}&current={{ current|collection_query_offset:pagelen:page }}{% endif %}{% endblock %}"{% if link_target %} target="{{ link_target }}"{% endif %}>
```

Comment:
This removes the `query_id=...` from the URL when if the collection object is not part of a query.